tests/postgres_tests/test_trigram.py

```diff
@@ -26,24 +27,36 @@ def test_trigram_search(self):
 
     def test_trigram_similarity(self):
         search = 'Bat sat on cat.'
-        self.assertQuerysetEqual(
-            self.Model.objects.filter(
-                field__trigram_similar=search,
-            ).annotate(similarity=TrigramSimilarity('field', search)).order_by('-similarity'),
-            [('Cat sat on mat.', 0.625), ('Dog sat on rug.', 0.333333)],
-            transform=lambda instance: (instance.field, instance.similarity),
-            ordered=True,
-        )
+        with connection.cursor() as cursor:
+            # extra_float_digits=0 is needed for consistent rounding on PostgresSQL <=12
+            cursor.execute('SET extra_float_digits TO 0')
```

Comment:
Changing DB parameters in tests is not something that we should do, IMO. We can rather use `round()`, e.g.
```python
# Round result of distance because PostgreSQL 12+ uses greater precision.
...
transform=lambda instance: (instance.field, round(instance.distance, 6)),
```

Reply:
Makes sense!